packages/lib/src/interactions/SelectToZoom.tsx

```diff
@@ -13,33 +12,20 @@ type Props = CommonInteractionProps;
 
 function SelectToZoom(props: Props) {
   const context = useAxisSystemContext();
-  const { visRatio } = context;
+  const { canvasSize, canvasRatio, visRatio } = context;
```

Comment:
`dataToWorld` may also destructured from `context`. 

Also,  keeping the `context` variable is no longer useful since `getVisibleDomains` was removed.